### PR TITLE
Add standard run scripts

### PR DESCRIPTION
**Pull request by IsaackMwenda**
This adds the standard run script except for fetch_raw_data.sh that was pushed in a different pr https://github.com/AfricasVoices/Project-WUSC-KEEP-II/pull/9

### CODE REVIEW

File: run_scripts/1_coda_get.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    exit
+fi
+
+AUTH=$1
+CODA_V2_ROOT=$2
+DATA_ROOT=$3
+
+./checkout_coda_v2.sh "$CODA_V2_ROOT"
+
+PROJECT_NAME="Wusc-Keep-II"
```
Comment (as2388) [inline run_scripts/1_coda_get.sh]: WUSC-KEEP

**Review by as2388 — APPROVED** [review]

**Comment by as2388** [comment]
Requested review from @lukechurch to confirm that we're pushing the demogs for each location to different datasets. Relevant code is here: https://github.com/AfricasVoices/Project-WUSC-KEEP-II/blob/f8abe74f429c4dd082eacac0101caea4be659f35/run_scripts/1_coda_get.sh#L18-L46

**Comment by lukechurch** [comment]
> Requested review from @lukechurch to confirm that we're pushing the demogs for each location to different datasets. Relevant code is here:
> 
> https://github.com/AfricasVoices/Project-WUSC-KEEP-II/blob/f8abe74f429c4dd082eacac0101caea4be659f35/run_scripts/1_coda_get.sh#L18-L46

I think this is an OK strategy for WUSC - we need quite a bit more separation that we do with multi-region CSAP projects

**Comment by lukechurch** [comment]
Is this ready for my review?

File: run_scripts/1_coda_get.sh (hunk 1 above)
Comment (IsaackMwenda) [reply run_scripts/1_coda_get.sh]: updated to WUSC-KEEP-II thanks.

**Comment by IsaackMwenda** [comment]
> Is this ready for my review?
Yes PTAL @lukechurch

**Review by lukechurch — APPROVED** [review]

**Pull request merged**